sequelize/index.d.ts

```diff
@@ -6160,4 +6160,6 @@ declare namespace sequelize {
 
 declare var sequelize: sequelize.SequelizeStatic;
 
-export = sequelize;
+declare module "sequelzie" {
```

Comment:
Should this be `declare module "sequelize" {` instead? This might have been a simple typo.